### ISSUE TITLE
Feature requesr: Add background color for iOS and Android

### ISSUE THREAD

**Issue by BorntraegerMarc** [apache/cordova-plugin-splashscreen#160]
I would like to request a feature to add the property `SplashScreenBackgroundColor` for iOS and Android for the reason outlined here: 
https://github.com/ionic-team/cordova-plugin-ionic/issues/151

**Comment by janpio** [comment]
Just do understand the issue better: Why isn't adding a background color to the splashscreen an option?

**Comment by BorntraegerMarc** [comment]
Thanks for the reply @janpio

Generally a background in the actual image could be added. But in our case we are using the same image in some places of the app (To save disk storage). And the background is only need for the splash screen.

**Comment by janpio** [comment]
Thanks for the clarification, although very special case it makes sense.

**Comment by jcesarmobile** [comment]
Windows platform (and maybe browser) already have a `SplashScreenBackgroundColor` preference.

On iOS would be as simple as adding a color to the image view where the splash is loaded, but it only applies to the legacy splash images, didn't check on the storyboard use case. Not sure about Android neither.

**Comment by IndieSW** [comment]
About one year ago, I added support on Android for `SplashScreenBackgroundColor` and `SplashScreenTheme` in a private fork but never submitted a pull request.  I'm not sure but I think the Android version of the plugin has been modified since then.

**Comment by janpio** [comment]
If you still have the fork, please still open the pull request - we can fix those conflicts if there are any. The important thing are your changes @IndieSW 👍

**Comment by IndieSW** [comment]
I'd be happy to but I don't have time right now to resolve conflicts and test.  I've used my fork in an app and tested though so I know the changes work.

**Comment by janpio** [comment]
Yeah, or whatever you already have. If you submit it and link to this issue, we can have a look and create a new fork based on your changes. Probably not that drastic.

**Comment by janpio** [comment]
I took your file and applied it to the 4.0.x release branch. Makes a much nicer diff: https://github.com/apache/cordova-plugin-splashscreen/pull/162/files

**Comment by IndieSW** [comment]
@janpio Perfect!

**Comment by Hadatko** [comment]
Any progress here?

**Comment by HansKrywaa** [comment]
Hey there, any updates? It's a topic for Dark Mode too.

**Comment by breautek** [comment]
Removing the android platform tag because cordova-android@11 splashscreen uses the Android 12 Splashscreen API, and exposes an option to set the background color.

I'm not sure if adding a background option is possible for iOS. I'm fairly certain you have to supply an image, but I'll leave this open for someone who is more knowledgeable in iOS development.

**Comment by erisu** [comment]
As this repo also does not support iOS anymore, I will be moving this ticket to the cordova-ios repo
